cli/mender-artifact/debugfs.go

```diff
@@ -33,10 +34,26 @@ func debugfsCopyFile(file, image string) (string, error) {
 	dumpCmd := fmt.Sprintf("dump %s %s", file,
 		filepath.Join(tmpDir, filepath.Base(file)))
 	cmd := exec.Command("debugfs", "-R", dumpCmd, image)
-	if err = cmd.Run(); err != nil {
+	ep, err := cmd.StderrPipe()
+	if err != nil {
+		return "", errors.Wrap(err, "failed to open stderr pipe of command")
+	}
+	if err = cmd.Start(); err != nil {
 		os.RemoveAll(tmpDir)
 		return "", errors.Wrap(err, "debugfs: run debugfs dump")
 	}
+	data, err := ioutil.ReadAll(ep)
+	if err != nil {
+		return "", errors.Wrap(err, "Failed to read from stderr-pipe")
+	}
+
+	if len(data) > 0 && strings.Contains(string(data), "File not found") {
+		return "", fmt.Errorf("file %s not found in image", file)
+	}
+	if err = cmd.Wait(); err != nil {
+		os.RemoveAll(tmpDir)
```

Comment:
Nice you added this, but what with other failed scenarios? We should probably remove the temp dir for all the error cases.